src/MetalsFeatures.ts

```diff
@@ -46,9 +46,8 @@ export class MetalsFeatures implements StaticFeature {
     (params.capabilities.experimental as any).inputBoxProvider = true;
     (params.capabilities.experimental as any).didFocusProvider = true;
     (params.capabilities.experimental as any).slowTaskProvider = false;
-    (
-      params.capabilities.experimental as any
-    ).executeClientCommandProvider = true;
+    (params.capabilities
+      .experimental as any).executeClientCommandProvider = true;
```

Comment:
can you try putting everything on a single line, and re-running prettier? I think the line break is "accidental"

Reply:
Yea I actually tried that, and it still breaks it even if I have it on the same line :/

Reply:
I also tried:

```typescript
     (params.capabilities.experimental as any)
       .executeClientCommandProvider = true;
```
However, even this is re-written.